tests/PYME/IO/test_countdir.py

```diff
@@ -1,4 +1,8 @@
 
+import pytest
+import sys
+
+@pytest.mark.xfail(sys.platform=='win32', reason='countdir.c not compiled on non-posix platforms')
```

Comment:
```suggestion
@pytest.mark.skipif(sys.platform=='win32', reason='countdir.c not compiled on non-posix platforms')
```

There's no point in trying on windows - I'd usually use xfail for something which *should* work but has some reason (e.g. lack of libraries on platform x) which is is making it fail in the test environment. If the test is not relevant, we should just skip it.